### ISSUE TITLE
Some problems about DIou-darknet-tiny.

### ISSUE THREAD

**Issue by y200504040u** [Zzh-tju/DIoU-darknet#13]
I trained my tiny version by the implement.
I added the ciou parameters in each yolo layer of yolo-tiny.cfg in the original Darknet based on the coco-ciou.cfg like this.
<img width="466" alt="yolo-tiny" src="https://user-images.githubusercontent.com/57352939/73645377-25c4d100-46b2-11ea-825e-c695fd2caaf2.png">
My training log like this:
<img width="1266" alt="yolo-tiny-02" src="https://user-images.githubusercontent.com/57352939/73645652-a683cd00-46b2-11ea-8e71-d469ebaae281.png">
I trained only person class.The mean iou is above 0.5.
I run inference on the test image by my weight, the result is  no box.
<img width="614" alt="image" src="https://user-images.githubusercontent.com/57352939/73646034-87d20600-46b3-11ea-9d16-04854d612a0a.png">
I wanted to make sure that I wasn't doing something wrong.

**Comment by Zzh-tju** [comment]
You want to open the picture on the server, right? The image is on yourpath/DIoU-Darknet/prediction.jpg

**Comment by y200504040u** [comment]
> You want to open the picture on the server, right? The image is on yourpath/DIoU-Darknet/prediction.jpg

Yes.I download the prediction.jpg from the server to my PC.The result is none.
The log is like this if I get result.
<img width="409" alt="image" src="https://user-images.githubusercontent.com/57352939/73650230-9f61bc80-46bc-11ea-8e66-537e19d64c4f.png">
I log the num of boxes like this.Then make
<img width="642" alt="image" src="https://user-images.githubusercontent.com/57352939/73651558-a0481d80-46bf-11ea-8a6e-247367e61e22.png">
But got none.
Maybe there is something wrong in network running.

**Comment by Zzh-tju** [comment]
can you show me your cfgfile？ @y200504040u

**Comment by y200504040u** [comment]
> can you show me your cfgfile？ @y200504040u

[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=256
subdivisions=8
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

## single gpu
#learning_rate=0.001
#burn_in=1000
#max_batches = 550400

## 2 gpu
#learning_rate=0.0005
#burn_in=2000
#max_batches = 500200

## 4 gpu
learning_rate=0.00025
burn_in=4000
max_batches = 500200
#learning_rate=0.001
#burn_in=1000
#max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear



[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
iou_normalizer=0.5
cls_normalizer=1.0
iou_loss=ciou
nms_kind=greedynms
beta1=0.6

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
iou_normalizer=0.5
cls_normalizer=1.0
iou_loss=ciou
nms_kind=greedynms
beta1=0.6

**Comment by Zzh-tju** [comment]
maybe you should test on another weight file

**Comment by y200504040u** [comment]
I'm sorry that I don't know how to add attachments in issues.

**Comment by y200504040u** [comment]
Could you give me your email? Thanks.

**Comment by Zzh-tju** [comment]
zh_zheng@tju.edu.cn
